dev-resources/puppetlabs/services/jruby/jruby_interpreter_test/puppet.conf

```diff
@@ -1,2 +1,2 @@
 [main]
-vardir = ./target/master-var
\ No newline at end of file
+vardir = ./target/master-var-jruby-int-test
```

Comment:
It's not an error, but sure....thought my editor actually puts these in so not sure why this doesn't have one :smile: